manager/src/main/resources/db/migration/0.1/V0.1.23__add_cleanup_processing_error_procedure.sql

```diff
@@ -0,0 +1,25 @@
+-- This procedure by choice does not "commit" at the end to avoid clashing
+-- with Hibernate transaction management.
+create procedure cleanup_processing_error(max_errors_per_bridge integer)
+    language plpgsql
+as
+$$
+declare
+    exceeding_bridge record;
+begin
+    for exceeding_bridge in
+        select bridge_id from processing_error group by bridge_id having count(*) > max_errors_per_bridge
```

Comment:
Not sure if we should consider this scenario here now? let me know what you think:
- what if a bridge along with its processors has been previously removed? and the number of records in the `processing_error` is less than `max_errors_per_bridge`? 
These records will never be removed since there is no FK associated with the `bridge` table and the number of records for that bridge will be less than the `max_errors_per_bridge`

Should we consider this scenario?

Reply:
Good catch @afalhambra 👍

Reply:
I don't think I am the one that should decide this.

@danielezonca @robpblake do we want to delete the errors when a bridge is deleted or do we need to keep them for some reason?

Reply:
@kostola +1 to delete them if the bridge is deleted for this implementation.

Reply:
@robpblake done :+1: